UltraStar Play/Assets/Common/UI/UiManager.cs

```diff
@@ -37,6 +38,11 @@ public static UiManager Instance
     [InjectedInInspector]
     public Tooltip tooltipPrefab;
 
+    private Subject<Vector3> mousePositionChangeEventStream = new Subject<Vector3>();
+    public IObservable<Vector3> MousePositionChangeEventStream => mousePositionChangeEventStream;
+
+    public bool DialogOpen => dialogs.Count > 0;
```

Comment:
when does this get evaluated? only when a new UiManager is created?

Reply:
The expression is evaluated every time the Property is accessed (=> is like a lambda).
It is equivalent to `public DialogOpen { get { return dialogs.Count > 0; } }`